src/main/kotlin/com/openlattice/organizations/ExternalDatabaseManagementService.kt

```diff
@@ -124,14 +124,20 @@ class ExternalDatabaseManagementService(
         return organizations.keys
     }
 
-    fun getExternalDatabaseTables(orgId: UUID): Set<OrganizationExternalDatabaseTable> {
-        return organizationExternalDatabaseTables.values(belongsToOrganization(orgId)).toSet()
+    fun getExternalDatabaseTables(orgId: UUID): Map<UUID, OrganizationExternalDatabaseTable> {
+        return organizationExternalDatabaseTables.entrySet(belongsToOrganization(orgId))
+                .map {
+                    it.key to it.value
+                }.toMap()
     }
 
-    fun getExternalDatabaseTablesWithColumns(orgId: UUID): Map<OrganizationExternalDatabaseTable, Set<OrganizationExternalDatabaseColumn>> {
+    fun getExternalDatabaseTablesWithColumns(orgId: UUID): Map<Pair<UUID, OrganizationExternalDatabaseTable>, Map<UUID, OrganizationExternalDatabaseColumn>> {
         val tables = getExternalDatabaseTables(orgId)
         return tables.map {
-            it to (organizationExternalDatabaseColumns.values(belongsToTable(it.id)).toSet())
+            Pair(it.key, it.value) to (organizationExternalDatabaseColumns.entrySet(belongsToTable(it.key))
```

Comment:
same thing here with skipping the `.map { .. } .toMap()`

Reply:
same thing again with having to move the map{}s to the controller.